ngraph/frontend/onnx_import/src/op/org.openvinotoolkit/experimental_detectron/detection_output.hpp

```diff
@@ -0,0 +1,64 @@
+//*****************************************************************************
+// Copyright 2017-2021 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include "ngraph/node.hpp"
+#include "ngraph/op/experimental_detectron_detection_output.hpp"
+#include "onnx_import/core/node.hpp"
+
+namespace ngraph
+{
+    namespace onnx_import
+    {
+        namespace op
+        {
+            namespace set_1
+            {
+                OutputVector experimental_detectron_detection_output(const Node& node)
+                {
+                    using DetectionOutput = ngraph::op::v6::ExperimentalDetectronDetectionOutput;
+
+                    auto inputs = node.get_ng_inputs();
+                    auto rois = inputs[0];
+                    auto deltas = inputs[1];
+                    auto scores = inputs[2];
+                    auto im_info = inputs[3];
+
+                    DetectionOutput::Attributes attrs{};
+                    attrs.score_threshold = node.get_attribute_value<float>("score_threshold");
+                    attrs.nms_threshold = node.get_attribute_value<float>("nms_threshold");
+                    attrs.max_delta_log_wh = node.get_attribute_value<float>("max_delta_log_wh");
+                    attrs.num_classes = node.get_attribute_value<std::int64_t>("num_classes");
+                    attrs.post_nms_count = node.get_attribute_value<std::int64_t>("post_nms_count");
+                    attrs.max_detections_per_image =
+                        node.get_attribute_value<std::size_t>("max_detections_per_image");
+                    attrs.class_agnostic_box_regression = static_cast<bool>(
+                        node.get_attribute_value<int64_t>("class_agnostic_box_regression"));
+                    attrs.deltas_weights =
+                        node.get_attribute_value<std::vector<float>>("deltas_weights");
```

Comment:
you should add default values for those attributes. You can get the values from model-optimizer/extensions/front/onnx/detectionoutput_ext.py.

Reply:
Done